content/2_Bronze/Intro_Sets.mdx

```diff
@@ -371,14 +427,22 @@ for (pair<int,int> x : m) {
 
 <JavaSection>
 
-To iterate over maps, you can use a `for` loop over the keys.
+To iterate over maps, you can use a for-each loop over the keys:
 
 ```java
 for (int k : m.keySet()) {
 	System.out.println(k + " " + m.get(k));
 }
 ```
 
+You can also use a for-each loop over the entries:
+
+```java
+for (Map.Entry entry : m.entrySet()) {
```

Comment:
oh uh
(optional) mention somewhere in java & c++ that like all these set/dict things are in `java.util` & `<set>` + `<map>` respectively

Reply:
I'm not sure what to say in c++ since I'm not too familiar with it, could you write out what that would look like?

Reply:
uh
in c++ `std::set` is in the header `<set>` and `std::map` is in the header `<map>`

Reply:
So like would I say unordered sets are implemented by std::unordered_set in the <set> header?

Reply:
uh unordered sets are actually in `<unordered_set>` lol

Reply:
don't see you mentioning `java.util` anywhere?

Reply:
mentioned on line 121

Reply:
i'll bellieve you